concrete-core/src/backends/core/implementation/entities/ggsw_ciphertext.rs

```diff
@@ -56,3 +61,61 @@ impl GgswCiphertextEntity for GgswCiphertext64 {
         self.0.decomposition_base_log()
     }
 }
+
+/// A structure representing a GGSW ciphertext with 64 bits of precision in the Fourier domain.
```

Comment:
it could be nice to have the same level of details than below in `FourierGgswCiphertext32`

Reply:
What do you mean? The details below say that despite the name, data are written on 64 bits, which could be misleading. Here everything's on 64 bits, so I'm not sure what to write.

Reply:
@damienligier do you have an answer here?

Reply:
below it is said in the doc this additional piece of information:
```rust
/// Note: even though the coefficients have 32 bits of precision in the standard domain,
/// they are represented with 64 bits of precision in the Fourier domain.
```

Reply:
@damienligier Yes, because there's a distinction between 32 and 64 bits. Here, everything's 64 bits, so what do you suggest that we add?